package.json

```diff
@@ -46,40 +46,38 @@
   "license": "MPL-2.0",
   "repository": "mozilla/send",
   "availableLanguages": [
-    "en-US",
```

Comment:
@dannycoates Are these available languages in a "specific" order? I think my script just blindly A-z the locales, but wasn't sure if having "en-US" at the front is important, or if it serves as a fallback locale.